pandas/core/generic.py

```diff
@@ -8327,24 +8327,95 @@ def _doc_parms(cls):
 """
 
 _cnum_doc = """
+Return %(desc)s over a DataFrame or Series axis.
+
+Returns a DataFrame or Series of the same size containing the %(desc)s.
 
 Parameters
 ----------
 axis : %(axis_descr)s
```

Comment:
Not sure were `axis_descr` is defined, but the format it `{0 or 'index', 1 or 'columns'}` if I'm not wrong. You can check recent merge PRs to be sure.